GitUI/CommandsDialogs/RepoHosting/ViewPullRequestsForm.cs

```diff
@@ -73,22 +73,27 @@ private void ViewPullRequestsForm_Load(object sender, EventArgs e)
             _loader.LoadAsync(
                 () =>
                 {
-                    var t = _gitHoster.GetHostedRemotesForModule().ToList();
-                    foreach (var el in t)
+                    var hostedRemotes = _gitHoster.GetHostedRemotesForModule().ToArray();
+
+                    // load all hosted repositories.
+                    foreach (var hostedRemote in hostedRemotes)
                     {
-                        el.GetHostedRepository(); // We do this now because we want to do it in the async part.
+                        try
+                        {
+                            hostedRemote.GetHostedRepository(); // We do this now because we want to do it in the async part.
+                        }
+                        catch (Exception)
```

Comment:
ditto

Reply:
I think here we'd want to let the user know that we couldn't reach certain remotes. Otherwise if the user won't see some of the expected remotes they may not be able to immediately reconcile it.
Perhaps show a task dialog-based message:
```
We couldn't fetch the following repositories:
  - https://bla/bla/bla.git
  - https://foo/foo/foo.git
```

Reply:
I did something like that first but it will be more difficult to achieve: the error is displayed twice because the method is called from 2 places and I was not able to understand if that's normal and what will be the consequences to remove the first call :(